app/controllers/api/v1/settings_controller.rb

```diff
@@ -0,0 +1,20 @@
+module Api
+  module V1
+    class SettingsController < Api::V1::ApiController
+      skip_before_action :authenticate_user!
+
+      def need_update
+        current_version = Gem::Version.new(params[:device_version])
+        min_version = Gem::Version.new(version) if version
+        must_update = min_version > current_version
+        render json: { 'must_update': must_update }
```

Comment:
I would use or `need_update` or `must_update` everywhere, so we use the same name at the endpoint and in the response, what do you think?

Reply:
I though it as the endpoint (need_update) is the question and the response (must_update)